docs/content/v2.20/reference/configuration/operating-systems.md

```diff
@@ -21,15 +21,15 @@ Unless otherwise noted, operating systems are supported by all supported version
 | :--------------- | :------------- | :------------- | :---- |
```

Comment:
Change the first sentence so it doesn't say 'are supported':
"The following table describes the operating system and architecture support for [deploying YugabyteDB](../../../deploy/manual-deployment/)."